src/evm_context.rs

```diff
@@ -175,6 +211,13 @@ fn calldata() -> Identifier {
     identifier("_calldata", 1026)
 }
 
+fn memory() -> Identifier {
+    identifier("_memory", 1027)
+}
+fn memory_type() -> &'static str {
+    "(Array (_ BitVec 256) (_ BitVec 8))"
```

Comment:
We should probably extract these into a trait and have modules for memory/storage/calldata at some point.